latex/headers/common-byval.h

```diff
@@ -19,17 +19,25 @@ class T {
   ...
 
  public:
-  // Rule of Zero: the following five special member functions should not be explicitly declared
+  // If any of the following five special member functions are not
+  
+  // public, inline or defaulted, then all five of them should be
+    
+  // explicitly declared (see rule of five).
+    
+  // Otherwise, none of them should be explicitly declared
+    
```

Comment:
Why these 4 empty lines above in between?

Reply:
There was a blank line between every line in the original.  I was just trying to be consistent, but it wasn't obvious what to do with multiline comments.  Anyway, I removed the blank lines.